external-builds/pytorch/build_prod_wheels.py

```diff
@@ -408,14 +408,21 @@ def do_build(args: argparse.Namespace):
     # on directory layout.
     # Obviously, this should be completely burned with fire once the root causes
     # are eliminted.
-    hip_device_lib_path = get_rocm_path("root") / "llvm" / "amdgcn" / "bitcode"
+    if is_windows:
```

Comment:
This was an unfortunate typo in the original. You can collapse both of these down to "root"/lib/llvm/amdgcn/bitcode and avoid the platform branch.

Really need to track down the bug that is causing this.

Reply:
Thanks, I suspected that we could collapse it but figured the symlink was there for "reasons". At least for code we control we can use the full path that is consistent.